src/lib/style-properties.html

```diff
@@ -181,6 +188,50 @@
         rule.cssText = output;
       },
 
+      // Apply keyframe transformations to the cssText of a given rule. The
+      // keyframeTransforms object is a map of keyframe names to transformer
+      // functions which take in cssText and spit out transformed cssText.
+      applyKeyframeTransforms: function(rule, keyframeTransforms) {
+        var input = rule.cssText;
+        var output = rule.cssText;
+
```

Comment:
Let's remove these empty lines. We generally don't space out function contents like this and prefer factoring instead.